src/main/resources/net.fabricmc.fabric.mixins.common.json

```diff
@@ -38,4 +39,4 @@
   "injectors": {
     "defaultRequire": 1
   }
-}
```

Comment:
Unnecessary change

Reply:
This is just git weirdness, really. Not much I can do about it.

Reply:
You added a new line to the end of the file (probably ran an auto-format?)

Reply:
Ahhh yeah, that'd be autoformat.